portal-impl/src/com/liferay/portlet/asset/service/impl/AssetVocabularyLocalServiceImpl.java

```diff
@@ -435,6 +443,39 @@ public AssetVocabulary updateVocabulary(
 		return vocabulary;
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public AssetVocabulary upsertVocabulary(
+			long userId, long groupId, String title,
+			Map<Locale, String> titleMap, Map<Locale, String> descriptionMap,
+			String settings, String externalReferenceId,
```

Comment:
I don't think this was intentional, but this is the only place where it is "externalReferenceId" instead of "externalReferenceCode".